README.md

```diff
@@ -6,76 +6,86 @@
+
+```bash
+kubectl apply configmap kubernetes-dasboard-certs --from-file=$HOME/certs -n kube-system
 ```
 
-If you are using Dashboard on a cluster with disabled authorization, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard-no-rbac
+Afterwards, you are ready to deploy Dashboard using following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
 ```
 
-You can also install unstable HEAD builds with the newest features that the team works on by
-following the [development guide](docs/devel/head-releases.md).
+### Alternative setup
 
-Note that for the metrics and graphs to be available you need to
-have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster.
+This setup is not fully secure, however it does not require any additional steps. In this setup access control can be
+ensured only by using [Authorization Header](
+https://github.com/kubernetes/dashboard/wiki/Access-control#authorization-header) feature. To deploy Dashboard execute
+following command:
 
-## Usage
-The easiest way to access Dashboard is to use kubectl. Run the following command in your desktop environment:
-```shell
-$ kubectl proxy
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/alternative/kubernetes-dashboard.yaml
 ```
-kubectl will handle authentication with apiserver and make Dashboard available at [http://localhost:8001/ui](http://localhost:8001/ui)
 
-The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
+**NOTE:** You can also install latest development builds with the newest features that the team works on by
+following the [Development Release](https://github.com/kubernetes/dashboard/wiki/Installation#development-release)
+guide.
 
-## Alternative Usage
-You may access the UI directly via the apiserver proxy. Open a browser and navigate to `https://<kubernetes-master>/ui`.
+**NOTE:** You need to have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster for the metrics
+and graphs to be available. You can read more about it in [Integrations](
+https://github.com/kubernetes/dashboard/wiki/Integrations) guide.
 
-Please note that this works only if the apiserver is set up to allow authentication with username and password or certificates, however certificates require some manual steps to be installed in the browser. This is not currently the case with the setup tool `kubeadm`. See [documentation](http://kubernetes.io/docs/admin/authentication/) if you want to configure it manually.
+## Usage
 
-If the username and password is configured but unknown to you, then use `kubectl config view` to find it.
+After installing Dashboard check [Accessing Dashboard](https://github.com/kubernetes/dashboard/wiki/Accessing-dashboard)
```

Comment:
This is important, but not prominent. Might be overlooked by users

Reply:
I'm not sure. How would you make it more prominent?